it/uk/gov/hmrc/fileupload/support/FakeFileUploadBackend.scala

```diff
@@ -7,44 +7,46 @@ import com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig
 import com.github.tomakehurst.wiremock.verification.LoggedRequest
 import io.findify.s3mock.S3Mock
 import io.findify.s3mock.request.CreateBucketConfiguration
-import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.time.{Millis, Seconds, Span}
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
 import org.scalatest.{BeforeAndAfterAll, Suite}
 import play.api.http.Status
 import uk.gov.hmrc.fileupload.{EnvelopeId, FileId}
 
 import scala.collection.JavaConverters._
-import scala.concurrent.ExecutionContextExecutor
 
-trait FakeFileUploadBackend extends BeforeAndAfterAll with ScalaFutures {
+trait FakeFileUploadBackend extends BeforeAndAfterAll with ScalaFutures with IntegrationPatience {
   this: Suite =>
 
-  implicit override val patienceConfig: PatienceConfig = PatienceConfig(timeout = Span(10, Seconds), interval = Span(200, Millis))
-  implicit val executionContext: ExecutionContextExecutor = scala.concurrent.ExecutionContext.Implicits.global
-
   lazy val backend = new WireMockServer(wireMockConfig().dynamicPort())
   lazy val backendPort: Int = backend.port()
 
-  lazy val workDir = s"/tmp/s3"
   // create and start S3 API mock
-  lazy val s3MockServer = S3Mock(port = 8001, dir = workDir)
+  lazy val s3Port = 8001
+  lazy val workDir = s"/tmp/s3"
```

Comment:
would we be better using java.io.tmpdir or createTempDirectory just in case /tmp isnt usable